maestro-cli/src/main/java/maestro/cli/device/DeviceService.kt

```diff
@@ -168,6 +177,217 @@ object DeviceService {
         }
     }
 
+    /**
```

Comment:
For all the device creation and env related functions when it comes to Android, do we know whether that works on Windows / Linux too?

Reply:
It should work, as we use standard tools from the Android SDK i.e  `cmdline-tools/latest/bin/avdmanager`

With that said, I have not tested in these environments. I can reach out to Manish to run a few command to validate

Reply:
Reached out to Manish to assist with testing

Reply:
According to Manish device listing never worked and by extension these updates will also not work. This is a [limitation](https://maestro.mobile.dev/getting-started/installing-maestro/windows#2884) with WSL + ANDROID_HOME.

What I propose is to detect if it's running on Windows and show a proper error, until we decide to properly support it.

I will also see if there's a quick solution for referencing ANDROID_HOME from WSL

Reply:
We tried to install sdk tools in the WSL environment but it still didn't work. 
Will check if we can show a more informative error but Windows will not work with this command.